common/constant/key.go

```diff
@@ -78,6 +78,7 @@ const (
 
 const (
 	DUBBOGO_CTX_KEY = "dubbogo-ctx"
+	CONTEXT_KEY     = "context"
```

Comment:
what is that key use for ? it should add some comment

Reply:
It's not used, so I remove it.